packages/plugins/ens/src/index.js

```diff
@@ -132,32 +132,40 @@ class ENS {
   }
 
   async addArtifactFile(_params, cb) {
-    this.getEnsConfig((err, config) => {
-      this.events.request("pipeline:register", {
-        path: [this.config.embarkConfig.generationDir, 'config'],
-        file: 'namesystem.json',
-        format: 'json',
-        content: Object.assign({}, this.embark.config.namesystemConfig, config)
-      }, cb);
-    });
+    if (this.embark.config.blockchainConfig.enabled) {
+      this.getEnsConfig((err, config) => {
```

Comment:
I think we should either check the `err` or put it as `_err` to show that it's useless